ydb/docs/en/core/concepts/_includes/ttl.md

```diff
@@ -176,6 +190,45 @@ The example below shows how to use the `modified_at` column with a numeric type
 
 {% endlist %}
 
+### Enabling eviction to Object Storage for an existing table {#enable-tiering-on-existing-tables}
```

Comment:
The "Concepts" documentation section is for high-level theoretical content. Old practical content is in the process of moving out. So, this content belongs to recipes and/or examples in YQL/SDK reference. Here, only give links there.